### ISSUE TITLE
Implement command GoCloud-CLI

### ISSUE THREAD

**Issue by lakindu95** [cloudlibz/gocloud#55]
Suggestion to build a command line interface for GoCloud

**Comment by amitsin6h** [comment]
I would like to work @lakindu95

**Comment by sladyn98** [comment]
@PratikDhanave Could this be a good issue to work on. I have a bit of experience with golang I could take this up.

**Comment by PratikDhanave** [comment]
@sladyn98  yes you can work on right now we want to integrate Azure cloud provider into gocloud you can also work on that.

**Comment by sladyn98** [comment]
@PratikDhanave yeah sure could you point the link to the issue and I can start working.
